akka-remote/src/main/scala/akka/remote/artery/compress/DecompressionTable.scala

```diff
@@ -27,6 +31,10 @@ private[artery] final case class DecompressionTable[T](originUid: Long, version:
 
 /** INTERNAL API */
 private[artery] object DecompressionTable {
+
+  val DisabledVersion: Byte = -1
+
   private[this] val _empty = DecompressionTable(0, 0, Array.empty)
   def empty[T] = _empty.asInstanceOf[DecompressionTable[T]]
```

Comment:
ok, empty is the starting table, so 0 makes sense, but then update the comment above: "-1 for empty table"